drivers/soundwire/intel.c

```diff
@@ -1722,6 +1722,9 @@ static int intel_resume_runtime(struct device *dev)
 	clock_stop_quirks = sdw->link_res->clock_stop_quirks;
 
 	if (clock_stop_quirks & SDW_INTEL_CLK_STOP_TEARDOWN) {
+
```

Comment:
nitpick: I wouldn't add empty lines at the beginning of `if () {}` and similar blocks - in this and following commits.

Reply:
I leave this as a degree of freedom, others use empty lines for for loops as well, it's just easier to identify the body and compare different cases.